From 04adde617fbda8a34c75cb88b3c8013e44e4cc14 Mon Sep 17 00:00:00 2001
From: Catalin Popescu <catalin.popescu@intel.com>
Date: Mon, 21 Mar 2016 15:52:44 +0100
Subject: [PATCH 4/7] Touch : add management of GPIO1 regulator.

Enable touchscreen driver to manage GPIO1 regulator
in power HAL callbacks.

Change-Id: Ic0a19bc212ee105ad87f7e64a143e71a1907d99e
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-21105
Signed-off-by: Catalin Popescu <catalin.popescu@intel.com>
Reviewed-on: https://android.intel.com:443/483018
---
 drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c b/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
index 245fc4e4..ee3ffa2 100644
--- a/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
+++ b/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
@@ -147,6 +147,7 @@ static struct synaptics_dsx_board_data s7300_board_data = {
 	.cap_button_map = &cap_button_map,
 	.vir_button_map = &vir_button_map,
 	.device_descriptor_addr = DSX_HID_DEVICE_DESCRIPTOR_ADDR,
+	.pwr_reg_name = "GPIO1",
 };
 
 static int do_i2c_transfer(struct i2c_client *client, struct i2c_msg *msg)
-- 
1.9.1

